static/js/popovers.js

```diff
@@ -404,6 +412,19 @@ function render_user_stream_list(streams, user) {
     });
 }
 
+function render_user_group_list(groups, user) {
+    groups.sort(compare_by_name);
+    const container = $("#groups .user-group-list");
```

Comment:
`#groups` seems like a really general selector that's asking for bugs -- the container element should be found via `#user-profile-modal`, and we should use the `#groups` ID for this widget.

Reply:
Hmm, this id is also used in toggling the tabs, so I did not remove the id but renamed it to `groups-tab`. And for container updated it so that we are now selecting it via `#user-profile-modal .user-group-list`.